packages/integration-karma/test/shadow-dom/Event-properties/Event.target.spec.js

```diff
@@ -55,3 +56,28 @@ if (!process.env.NATIVE_SHADOW) {
         });
     });
 }
+
+describe('When event target is accessed async', () => {
+    if (!process.env.NATIVE_SHADOW) {
```

Comment:
Should this conditional move outside the `describe`?

Reply:
we should probably add more tests for the other cases... inside this describe... the if can probably stay around that particular test